pkg/agent/endpoints/workload/handler.go

```diff
@@ -129,7 +132,7 @@ func (h *Handler) FetchJWTSVID(ctx context.Context, req *workload.JWTSVIDRequest
 }
 
 // FetchJWTBundles processes request for JWT bundles
-func (h *Handler) FetchJWTBundles(req *workload.JWTBundlesRequest, stream workload.SpiffeWorkloadAPI_FetchJWTBundlesServer) error {
+func (h *Handler) FetchJWTBundles(_ *workload.JWTBundlesRequest, stream workload.SpiffeWorkloadAPI_FetchJWTBundlesServer) error {
```

Comment:
nit: can you exclude this change? if we ever add a request parameter later on we will have to go back and change this again, and i don't really see a good reason to make the argument inaccessible to the handler method